docs/devices-api/api-reference/connect/connect-rpm-device.mdx

```diff
@@ -40,12 +34,7 @@ const metriport = new MetriportDevicesApi("YOUR_API_KEY");
 
 const token = await metriport.getConnectToken("metriportUserId");
 
-await metriport.connectProvider(
-  "tenovi", 
-  token, 
-  "device-id-0001", 
-  "85300"
-);
+await metriport.connectProvider("tenovi", token, "1234-0001,5678-0002", "85300");
```

Comment:
This will make the code component have horizontal scroll, I suggested on other PRs to keep those using as few columns as possible.

Reply:
Fixed 👍